### PR TITLE
Run tests inside Mono environment again.

### PR DESCRIPTION
**Pull request by aidmsu**
We can again test Hangfire inside Mono environment after [nuget issue 3085](https://github.com/NuGet/Home/issues/3085) issue was closed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,27 +31,22 @@ branches:
...
 script:
-  - travis_retry dotnet restore --disable-parallel
-  - dotnet test tests/Hangfire.Core.Tests -verbose
- 
-# Tests on Mono disabled – waiting for https://github.com/NuGet/Home/issues/3085
-# - travis_retry mono .nuget/NuGet.exe install .nuget/packages.config -OutputDirectory packages
-# - travis_retry mono .nuget/NuGet.exe install .
-# - xbuild /p:Configuration=Release /verbosity:minimal Hangfire.Mono.sln
-# - mono ./packages/xunit.runner.console.2.2.0-beta2-build3300/tools/xunit.console.exe ./tests/Hangfire.Core.Tests/bin/Release/Hangfire.Core.Tests.dll
+  - if test "$NET_PLATFORM" == "mono";
```
Comment (odinserj) [inline .travis.yml]: I think it's better to remove these variables and test existence of `mono` or `dotnet` commands, please see http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script.

File: .travis.yml [hunk 2]
```diff
@@ -35,27 +31,22 @@ branches:
 # Make sure build dependencies are installed.
 before_install:
```
Comment (odinserj) [inline .travis.yml]: Maybe it's better to rename this step to the `install`? Because now we don't have the latter.

**Comment by aidmsu** [comment]
@odinserj I removed extra instructions from .travis.yml. We will have to use last version of dotnet when Hangfire move to new csproj format.

**Comment by odinserj** [comment]
How clean, can't wait for conversion to the new csproj format 😄

**Pull request merged**